saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/registration/RelyingPartyRegistration.java

```diff
@@ -542,6 +549,15 @@ public Saml2MessageBinding getSingleSignOnServiceBinding() {
 			return this.singleSignOnServiceBinding;
 		}
 
+		/**
+		 * Return the list of preferred signature algorithm URIs, in preference order.
+		 * @return the list of signature algorithm URIs
+		 * @since 5.5
+		 */
+		public List<String> getSignAlgorithmUris() {
```

Comment:
I realize that OpenSAML calls this a signing algorithm, but since the metadata element looks like:

```xml
<alg:SigningMethod MinKeySize="256" Algorithm="http://www.w3.org/2001/04/xmldsigmore#ecdsa-sha256"/>
```

it would be more appropriate to name this `getSigningMethodAlgorithms`.